lib/auth/password_test.go

```diff
@@ -226,76 +226,167 @@ func (s *PasswordSuite) TestChangePasswordWithOTP(c *C) {
 	c.Assert(err, IsNil)
 }
 
-func (s *PasswordSuite) TestChangeUserAuthenticationPasswordOnly(c *C) {
+func (s *PasswordSuite) TestChangeUserAuthentication(c *C) {
```

Comment:
Could we move this test out of gocheck? No need to do it in this PR, but please leave a TODO.